content/docs/replication/replication-actions.md

```diff
@@ -61,3 +61,14 @@ While performing site-specific actions, please consult each driver's documentati
 
 For a brief guide on using actions for various DR workflows, please refer to this [document](../disaster-recovery). 
 
+### Deleting Replication Groups
+
+To delete a replication group from a cluster, the group must be empty. This
+means that all associated replication pairs must have been deleted/removed.
+
+This command will delete the empty replication group from the cluster and the
+associated replication consistency group from the PowerFlex array.
```

Comment:
How about 'associated **replication group** from the **backend** array' ?

Reply:
Good catch, typo from copying it over.